compiler/quilt/tools/command.py

```diff
@@ -1234,24 +1234,36 @@ def update(pkginfo, content):
 #     if pathlist[-1].startswith('x_'):
 #         pathlist[-1] = '-' + pathlist[-1][2:]
 #     return pathlist
-def export(pkginfo, output_path='.', filter=None, filename_mapper=lambda x: x,
-              force=False):
-    """Export package file data.  Note: this is *not* the opposite of import.
+def export(package, output_path='.', filter=lambda x: True, filename_mapper=lambda x: x):
+    """Export package file data.
 
-    :param package: package name, e.g., user/foo
+    Exports children of specified node to files (if they have file data).
+    Does not export dataframes or other non-file data.
+
+    Node path lists take the form ['foo', 'bar', 'baz'], and refer to a
+    specific node (such as foo.bar.baz) to be exported.
+
+    The `filter` function takes a node path list and should return `True` or
+    `False` to indicate the node's inclusion in the export.
+
+    The `filename_mapper` function takes a node path list, and should return
+    return a node path list indicating any name changes that are desired
+    for the export.  The altered nodepath will be reflected in the exported
+    filename, but will not affect any other resource.
+
+    :param package: package or subpackage name, e.g., user/foo or user/foo/bar
     :param output_path: distination folder
-    :param filter: Not implemented
-    :param filename_mapper:
-    :param force: if True, overwrite existing files
+    :param filter: function -- takes a node path list, returns True to export
+    :param filename_mapper: function -- takes and returns a node path list
     """
-    # TODO: filters
-    # TODO: tests
-    # TODO: export via symlinks / hardlinks (unwise / messing with datastore? windows compat?)
-    if filter is not None:
-        raise NotImplemented("filters not implemented yet")
+    # TODO: Update docs
+    # TODO: force
```

Comment:
bad merge?  I added force already (and needed it!)

Reply:
Bad merge maybe.  I'll kill this PR and retrace my steps.

Reply:
At the time it was created, it worked and was based on master, but moving it to tensorflow_virtual_files and the merge between our code there killed it.